lib/FormBody.php

```diff
@@ -175,16 +190,20 @@ private function determineContentType() {
     }
 
     public function getBodyLength(): Promise {
+        if ($this->cachedLength) {
+            return $this->cachedLength;
+        }
```

Comment:
This needs to be reset somewhere.

Reply:
These were reset in `resetCache()`, but never used anywhere. Leftover from refactoring?

Reply:
Maybe.